frontend/islands/islands.css

```
@@ -1,605 +1,448 @@
-
-.islands-hero-cta .primary {
-  background: var(--gradient-saffron-gold);
-  color: #1a1200;
-}
-
-.islands-hero-cta .primary:hover {
-  box-shadow: var(--shadow-glow-active);
-  transform: translateY(-2px);
-}
+.islands-page h1, .islands-page h2, .islands-page h3, .islands-page h4 { font-family: var(--font-display); font-weight: 700; letter-spacing: 0.01em; }
+.sr-only { position: absolute; width: 1px; height: 1px; padding: 0; margin: -1px; overflow: hidden; clip: rect(0,0,0,0); border: 0; }
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)